packages/superset-ui-core/src/chart/models/ChartMetadata.ts

```diff
@@ -1,3 +1,5 @@
+import { Behavior } from '../types/Base';
```

Comment:
Would prefer this to be called `ChartFilterBehavior` (after first-time seeing it in https://github.com/apache/superset/pull/13052 and being confused of what it is), but looks like I'm late to the party...

Reply:
It won't necessarily be restricted to filtering behavior, for example drill-downs which are essentially groupbys, but `ChartBehavior` could also be ok. Let me know if you want to update this now, otherwise we'll stick to this naming.

Reply:
Let's save the update to when we need to expand this functionality.